praw/models/helpers.py

```diff
@@ -3,12 +3,20 @@
 
 from ..const import API_PATH
 from .base import PRAWBase
+from .reddit.live import LiveThread
 from .reddit.multi import Multireddit, Subreddit
 
 
 class LiveHelper(PRAWBase):
     """Provide a set of functions to interact with LiveThreads."""
 
+    def __call__(self, id):  # pylint: disable=invalid-name,redefined-builtin
+        """Return a new lazy instance of :class:`~.LiveThread`.
+
+        :param id: A live thread ID, e.g., ``ukaeu1ik4sw5``.
```

Comment:
I know the other helpers don't yet have it, but could we provide an example of usage here so that people don't try to do `LiveHelper(reddit).__call__(id)`? This is something I've seen.

Adding something like the following should go a long way to make these docstrings, which are generated into RTD significantly more usable:

```rst

This method is intended to be used as:

.. code:: python
   livethread = reddit.live('ukaeu1ik4sw5')
```